tests/automation_framework/data/worker/workerregister_signingAlgorithm_RSA_OAEP_3072.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Rename the file to workerregister_signingAlgorithm_RSAOAEP3072.json, the last part should be one word as per the naming conventions.